package.json

```diff
@@ -12,12 +12,17 @@
 		"build": "npm run build-clean && npm run build-compile",
 		"build-clean": "rm -rf dist",
 		"build-compile": "tsc",
-		"doc": "typedoc --gitRevision master --ignoreCompilerErrors --out doc src/**/* --mode file",
+		"coverage-system-test": "DEBUG=app* nyc --nycrc-path=systemtests/.nyc.json --report-dir=coverage mocha --config systemtests/.mocha.json",
```

Comment:
Up to you: I have been doing `coverage-unit` and `coverage-system` accidentally and I actually think that's better. At the least it matches up with the test commands so its easier to swap between.

Reply:
I agree that this makes more sense. Happy to update it.